packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -7,6 +7,65 @@ import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  testWidgets('SliverReorderableList works well when having gestureSettings', (WidgetTester tester) async {
```

Comment:
How does this catch the regression? The tester will have the same default touch slop value since it is not run on android - you'd need to override the platform config on the test window to provide a touch slop value that is different than the default value.

Reply:
I have overridden the `MediaQueryData` of the pump widget.